next.config.js

```diff
@@ -2,9 +2,12 @@ const withCSS = require('@zeit/next-css');
 
 module.exports = withCSS({
   publicRuntimeConfig: {
-    NEXT_PUBLIC_EGO_API_ROOT: process.env.NEXT_PUBLIC_EGO_API_ROOT,
-    NEXT_PUBLIC_EGO_CLIENT_ID: process.env.NEXT_PUBLIC_EGO_CLIENT_ID,
-    EGO_PUBLIC_KEY: process.env.EGO_PUBLIC_KEY,
+    NEXT_PUBLIC_AUTH_API_ROOT: process.env.NEXT_PUBLIC_AUTH_API_ROOT,
+    NEXT_PUBLIC_AUTH_CLIENT_ID: process.env.NEXT_PUBLIC_AUTH_CLIENT_ID,
+    NEXT_PUBLIC_AUTH_REDIRECT_URI: process.env.NEXT_PUBLIC_AUTH_REDIRECT_URI,
+    NEXT_PUBLIC_AUTH_SCOPES: process.env.NEXT_PUBLIC_AUTH_SCOPES,
+    NEXT_PUBLIC_ARRANGER_API_URL: process.env.NEXT_PUBLIC_ARRANGER_API_URL,
```

Comment:
Just noticed this line, pulling your branch over mine. Looks like your PR didn't merge in the latest version of develop, would've caught this problem ahead of time. @anncatton 
<img width="251" alt="image" src="https://user-images.githubusercontent.com/2107110/141522037-564a56d3-3ef6-4021-b03a-56dd84f2fc87.png">

Reply:
so, we don't need the project_id or the api_url vars?

Reply:
oh now i see your pic.

Reply:
which makes me wonder... jenkins didn't fail the PR build, as it should've 🤔 
something to look into